spec/schemas/security._common.yaml

```diff
@@ -183,60 +244,232 @@ components:
           type: array
-          items:
-            type: string
-        disabled_transport_categories:
+          description: List of available sign-in options available.
+        password_validation_error_message:
+          type: string
+          description: Error message when password validation fails.
+        password_validation_regex:
+          type: string
+          description: Reg-ex to be used to perform password validation.
+
+    DistinguishedNames:
+      type: object
+      properties:
+        nodes_dn:
           type: array
           items:
             type: string
-        log_request_body:
+
+    DistinguishedNamesMap:
+      type: object
+      additionalProperties:
+        $ref: '#/components/schemas/DistinguishedNames'
+
+    DynamicConfig:
+      type: object
+      properties:
+        dynamic:
+          $ref: '#/components/schemas/DynamicOptions'
+
+    DynamicOptions:
+      type: object
+      properties:
+        filteredAliasMode:
+          type: string
+        disableRestAuth:
           type: boolean
-        resolve_indices:
+        disableIntertransportAuth:
           type: boolean
-        resolve_bulk_requests:
+        respectRequestIndicesOptions:
           type: boolean
-        exclude_sensitive_headers:
+        kibana: {}
+        http: {}
+        authc: {}
+        authz: {}
+        authFailureListeners: {}
+        doNotFailOnForbidden:
           type: boolean
-        enable_transport:
+        multiRolespanEnabled:
           type: boolean
-        enable_rest:
+        hostsResolverMode:
+          type: string
+        doNotFailOnForbiddenEmpty:
           type: boolean
-    UsersMap:
+
+    GenerateOBOToken:
       type: object
-      additionalProperties:
-        $ref: '#/components/schemas/User'
-    UserAttributes:
+      properties:
+        user:
+          type: string
+          description: The name of the entity requesting token.
+        authenticationToken:
+          type: string
+          description: The generated OBO token.
+        durationSeconds:
+          type: string
+          description: The duration of the token, defaulted to 300s.
+
+    HealthInfo:
       type: object
-      additionalProperties:
-        type: string
-    DistinguishedNamesMap:
+      properties:
+        message:
+          type: string
+        mode:
+          type: string
+        status:
+          type: string
+
+    IndexPermission:
       type: object
-      additionalProperties:
-        $ref: '#/components/schemas/DistinguishedNames'
-    DistinguishedNames:
+      properties:
+        index_patterns:
+          type: array
+          items:
+            type: string
+        dls:
+          type: string
+        fls:
+          type: array
+          items:
+            type: string
+        masked_fields:
+          type: array
+          items:
+            type: string
+        allowed_actions:
+          type: array
+          items:
+            type: string
+
```

Comment:
Why not!

Reply:
Actually, does this mess with editor folding of nodes? I mean in VSCode for example does it help or distract?

Reply:
Actually with IDE this doesn't matter much. This is a very low priority though. @DarshitChanpura What IDE do you use and why do you prefer adding an empty line?

Reply:
I use VSCode and IntelliJ interchangeably. I added this on my own (accidentally ?!) when I was trying to sort the entries lexicographically. I think it would be beneficial to add a new-line between entries.

Talking about folding in VSCode and IDEA, I think it works just fine:

| VScode | IDEA | 
|---------| ----- |
![Screenshot 2024-05-06 at 11 50 53 AM](https://github.com/opensearch-project/opensearch-api-specification/assets/35282393/7a4f7091-09ff-43c0-882f-c4a8d30cb8f3) | <img width="486" alt="Screenshot 2024-05-06 at 11 52 03 AM" src="https://github.com/opensearch-project/opensearch-api-specification/assets/35282393/57c45742-d507-47bf-a707-89ceb68b976d">